testing/run-write-tests.rst

```diff
@@ -5,6 +5,14 @@
 Running and writing tests
 =========================
 
+.. raw:: html
+
+    <script>
+    document.addEventListener('DOMContentLoaded', function() {
+      activateTab(getOS());
+    });
+    </script>
+
```

Comment:
I wonder if it's better to include this in every page that uses tabs or just once at the top level:
* if we do it for each page, we need to duplicate the function call for each page, and remember to add this whenever OS-specific tabs are added to a page.
* if we do it at the top level, it will be executed for each page, even when not needed

Another option would be contributing this upstream, and add a flag to enable/disable it.  It should also possible to include this snippet of HTML in an rst `|substitution|` to avoid code duplication, even if it might be overkill.

FWIW I tested on Linux/Firefox and I get "Unix" tabs (my `window.navigator.platform` is `"Linux x86_64"`).

Reply:
Yeah, I wasn't sure about including for all or just the required pages. It probably doesn't matter much either way, it's only a small bit of JS. 

I thought about contributing upstream, but tabs can be about anything, not only operating systems. Although as you mention, a flag would help. Another complication is the OS->tab mappings. For example, we have "Unix" shown for Linux.

Thanks for testing Linux!